Django/BookLibrary/authentication/serializers.py

```diff
@@ -0,0 +1,24 @@
+from rest_framework import serializers
+from django.contrib.auth.models import User
+
+'''This class is used to serialize the user deatils'''
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model=User
+        fields = ['id', 'username', 'email', 'password', 'first_name', 'last_name']
```

Comment:
In this case, API will return password also in response which is not correct. we should not expose password in API.

read about write_only fields in Serializer.